builtin/common/misc_helpers.lua

```diff
@@ -195,6 +195,27 @@ function string.split(str, delim, include_empty, max_splits, sep_is_pattern)
 	return items
 end
 
+--------------------------------------------------------------------------------
+function string.tail(str, num_lines)
```

Comment:
Not very important, but if we're gonna expose this, I'd probably try to give it a clearer name ("tail" may also be in reference to characters rather than lines). I'd also probably implement this using a reverse numeric for: `for index = #str, 1, -1 do local char = str:sub(i, i)`.